src/main/java/io/leonis/zosma/ipc/peripheral/component/Playstation.java

```diff
@@ -0,0 +1,35 @@
+package io.leonis.zosma.ipc.peripheral.component;
+
+/**
+ * The class Playstation.
+ *
+ * Describes the buttons on a playstation controller.
+ *
+ * @author Jeroen de Jong
+ */
+public class Playstation {
+
+  private Playstation() {
+  }
+
+  public interface RightClusterSupplier {
```

Comment:
I think these can be nested inside the corresponding Controller class

Reply:
What do you mean?